app/controllers/benchmark_runs_controller.rb

```diff
@@ -31,6 +31,10 @@ def create
     benchmark_run.result = params[:benchmark_run][:result]
     benchmark_run.save!
 
+    if params[:commit_hash]
+      BenchmarkRegressionJob.perform_later(benchmark_run, initiator, benchmark_type, benchmark_result_type, benchmark_type_params[:category])
```

Comment:
Please read this and see what can be improved here.

https://github.com/mperham/sidekiq/wiki/Best-Practices#1-make-your-job-parameters-small-and-simple